compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.h

```diff
@@ -31,6 +31,9 @@ enum class PipelinePhase {
   Start,
   // Runs the transform pipeline up to executable sources (pre translation).
   ExecutableSources,
+  // Runs the transform pipeline up to executable configurations (before
+  // strategy selection).
```

Comment:
"before translation strategy selection" to be clear and consistent with the following.